src/components/icons/model.icons.ts

```diff
@@ -12,6 +12,8 @@ const iconsSchema = new Schema({
   category: { type: Schema.Types.Mixed, required: true },
   type: { type: String, required: true },
   date: { type: String, required: true },
+  svgModifiedAt: String,
```

Comment:
in the interface these fields are optional, can you make sure that doesnt cause any problem as these can be undefined as well?